models/org.go

```diff
@@ -366,6 +367,39 @@ func getOwnedOrgsByUserID(sess *xorm.Session, userID int64) ([]*User, error) {
 		Find(&orgs)
 }
 
+// HasOrgVisible tells if the given user can see one or more of the given orgs
+func HasOrgVisible(orgs []*User, user *User) bool {
```

Comment:
We don't need `orgs` but only `org` or we should return`[]bool`.

Reply:
Done. This was used to see if any of the orgs that a user belongs to can be seen when viewing the profile page. There is additional logic in the template to hide org if user shouldn't see it.